pkg/domain/infra/abi/images.go

```diff
@@ -146,8 +146,27 @@ func (ir *ImageEngine) History(ctx context.Context, nameOrID string, opts entiti
 }
 
 func (ir *ImageEngine) Mount(ctx context.Context, nameOrIDs []string, opts entities.ImageMountOptions) ([]*entities.ImageMountReport, error) {
-	if opts.All && len(nameOrIDs) > 0 {
+	listMountsOnly := false
+	var images []*libimage.Image
+	var err error
```

Comment:
It seems to me this forward declaration can now be removed; `err` is scoped to the individual cases.

Reply:
… also in the other copy.

Reply:
The variable `err` must be declared because the compiler has a problem: `images declared and not used`. I don't know how to mix `:=` and `=`.

Reply:
Ah, OK.

It would be:
- move the `var err error` inside the switch cases
- or
  ```go
  images_, err := …
  if err != nil { …}
  images = images_
  ```

and neither is really elegant, let’s leave it as it is.